src/ol_infrastructure/applications/mitx/__main__.py

```diff
@@ -0,0 +1,45 @@
+from pulumi import StackReference, export
+from pulumi_aws import route53
+
+from ol_infrastructure.components.aws.s3_cloudfront_site import (
+    CloudfrontPriceClass,
+    S3ServerlessSite,
+    S3ServerlessSiteConfig,
+)
+from ol_infrastructure.lib.ol_types import Apps, AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+
+fifteen_minutes = 60 * 15
+dns_stack = StackReference("infrastructure.aws.dns")
+stack_info = parse_stack()
+mitx_environment = f"mitx-{stack_info.env_suffix}"
+aws_config = AWSBase(
+    tags={
+        "OU": "mitx",
+        "Environment": mitx_environment,
+        "Application": Apps.mitx_edx.value,
+    }
+)
+
+mitx_zone_id = dns_stack.get_output("mitx_zone_id")
+mfe_app_learning_config = S3ServerlessSiteConfig(
+    site_name=f"mfe-app-learning-{mitx_environment}",
+    domains=[f"mfe-app-learning-{mitx_environment}.mitx.mit.edu"],
+    bucket_name=f"mitx-mfe-app-learning-{mitx_environment}",
```

Comment:
This shouldn't be specific to the learning MFE, we should have one bucket and distribution to accommodate all of the MFEs that we deploy